### DISCUSSION TITLE
Playlists not starting properly

### DISCUSSION THREAD

**Discussion by krisnoble** [music-assistant/support#1621] [Q&A]
Not 100% sure if this is an issue with MA or just a quirk of my setup, but since b74 my playlists have not been starting properly so wondering if anyone can replicate?

My setup is:

- RPi 3a's running HifiberryOS
- HAOS + MA addon
- Players grouped in a UGP but in this case targeting single players
- All media and playlists stored in the /media directory

What seems to be happening is the playlists load, but don't actually play - they need the play button pressing manually either in HA or MA UI. This happens both when triggering via HA or via the blue play button in the MA UI.

If I run the command again when the player is in the stuck state, it works. So I'm working around it for now in HA by adding a second command after a 1s delay which seems to work reliably. Weirdly, running a "play" command after 1s in the same script/automation doesn't work although running the same play command manually does work, I can't figure out why.

Not seeing anything in the logs set at INFO globally, for the music controller and the queues controller.

**Comment by OzGav** [comment]
Does the queue get filled but just not start playing?

**Reply by krisnoble** [reply]
Just tested it, yes that's correct. The queue gets cleared and replaced with the right playlist, it just doesn't actually play.

**Reply by OzGav** [reply]
Yeah I have seen this before just queuing a playlist from within MA. Not sure why as there wasn’t anything in the logs there either. I’ll move this to an issue so it can get looked at, at some point…

**Reply by OzGav** [reply]
Sorry @krisnoble I don't think I elevated this to an issue. Is this still a problem?

**Reply by krisnoble** [reply]
No apologies necessary, anyway you did (#1624) and all sorted thanks so will close 👍🏻
